repour/server/endpoint/internal_scm_gitlab.py

```diff
@@ -82,6 +71,11 @@ async def internal_scm_gitlab(spec, repo_provider):
     return result
 
 
+def read_token(token_filepath):
+    with open(token_filepath, "r") as token_file:
+        return token_file.read()
```

Comment:
You might want to do:
```python
return token_file.read().strip()
```

to remove any newline at the end of the token when read in the file.

Reply:
meh if that's the only issue I could find, I'll just merge it and create another PR for it.